### DISCUSSION TITLE
Allow EE Update via Control Panel without disk_free_space()

### DISCUSSION THREAD

**Discussion by igreil** [ExpressionEngine/ExpressionEngine#3549] [Feature Requests]
### Suggest an idea for ExpressionEngine

Attempting to upgrade via the CP for me results in the following error:

```
Call to undefined function ExpressionEngine\Library\Filesystem\disk_free_space()
ee/ExpressionEngine/Library/Filesystem/Filesystem.php:831
```

Confirming via `phpinfo()`, the server has indeed disabled the `disk_free_space` (for "security reasons" apparently), so this is clearly not EE's fault –– hence an FR instead of a Bug Report.

Still, it would be nice to get an option to proceed with the update at this point: perhaps allowing to continue after a warning, or being able to disable this check via, say, a config variable altogether? Open to all reasonable suggestions … 

I know I have sufficient disk space, my budget host just doesn't allow me to see how much :-)

The pre-flight check is not necessary in most cases, and while prudent probably not strictly needed: after patching `Preflight.php` (to disable this check) I was able to proceed without further issues.

**Comment by intoeetive** [comment]
We're aiming to get this implemented with https://github.com/ExpressionEngine/ExpressionEngine/pull/4097
